docs/Contributors_Guide/github_workflow.rst

```diff
@@ -142,70 +140,72 @@ local repository.
     git merge origin develop
 
   The first command pulls changes from the original repository (the
-  `workflow-tools GitHub repository <https://github.com/ufs-community/workflow-tools>`_
+  `workflow-tools GitHub repository <https://github.com/ufs-community/workflow-tools>`__
   that you see when you run *git remote -v* and that you set to upstream in
-  step 4 above).  The second command pushes those changes to your forked
-  repository.  The third command will merge the local develop branch into
+  step 4 above). The second command pushes those changes to your forked
+  repository. The third command will merge the local 'develop' branch into
   the feature branch.
 
+   .. COMMENT: What does "step 4 above" reference? 
+      I don't see anything saying how to set an upstream remote. 
```

Comment:
We might want to borrow the rest of this explanation from MetPlus. In general, we won't require our internal developers to fork (they can just branch and push since they have write permissions). Any external contributors will need a fork, and might need that information. 

I think we should work through that later, though. Is it cool just to leave this comment here and add an Issue?

Reply:
@christinaholtNOAA Fine by me! I think we are submitting the RTD link, and the comments will be invisible on that. I will create an issue though.

Reply:
Issue #137 created: https://github.com/ufs-community/workflow-tools/issues/137